sefaria/model/text.py

```diff
@@ -2032,7 +2032,7 @@ def __init__(self, oref, lang="en", vtitle=None, exclude_copyrighted=False, actu
         self.is_merged = False
         self.sources = []
 
-        if self.lang not in self._oref.index_node.supported_languages:
+        if not self._oref.index_node.parent.supports_language(self.lang):
```

Comment:
I believe we don't want `.parent` here. This should run directly on `index_node`.